### DISCUSSION TITLE
😽 kitty-scrollback.nvim: v3.0.0 defaults to Neovim icon + adds support for new Neovim colorscheme

### DISCUSSION THREAD

**Discussion by mikesmithgh** [mikesmithgh/kitty-scrollback.nvim#130] [Announcements]
[kitty-scrollback.nvim v3.0.0](https://github.com/mikesmithgh/kitty-scrollback.nvim/releases/tag/v3.0.0) is officially released!

# What is kitty-scrollback.nvim?

A Neovim plugin (and Kitty Kitten) that allows you to navigate your Kitty scrollback buffer to quickly search, copy, and execute commands in Neovim.

![kitty_scrollback_screencapture_00_kitty_scrollback_nvim](https://github.com/mikesmithgh/kitty-scrollback.nvim/assets/10135646/5ec633b0-2d7c-4dc6-bd18-3e2a3eb5fb62)

Check out [Advanced Configuration](https://github.com/mikesmithgh/kitty-scrollback.nvim/wiki/Advanced-Configuration) for more demos.

# Migrating to v3.0.0
See [Migrating to v3.0.0](https://github.com/mikesmithgh/kitty-scrollback.nvim#-migrating-to-v300) for the detailed migration guide.

# What changed?
- The Neovim icon is now the default icon in the status window.
- Added support for the new Neovim default colorscheme (available on Neovim nightly `v0.10.0`)
- Added a lot of tests
- New logo on the README 

![ksb-new-logo-small](https://github.com/mikesmithgh/kitty-scrollback.nvim/assets/10135646/75716950-1758-4c89-ba95-f2e3bc8309a2)

- If you missed it, check out my ⚡️ Lightning Talk @ Neovim Conf 2023 [Developing kitty-scrollback.nvim](https://youtu.be/9TINe0J9rNg?si=_ISKqAQVS2NCuSRF)!

[![Developing kitty-scrollback.nvim NeovimConf 2023](https://github.com/mikesmithgh/kitty-scrollback.nvim/assets/10135646/d103a326-dfac-4948-8f15-d2885e646a6c)](https://www.youtube.com/watch?v=9TINe0J9rNg)

If you have any questions, comments, or feedback feel free to create an [issue](https://github.com/mikesmithgh/kitty-scrollback.nvim/issues) or open a [discussion](https://github.com/mikesmithgh/kitty-scrollback.nvim/discussions).

**Comment by gregorias** [comment]
Amazing. Congrats on another major release.
